### PR TITLE
add settings for signature and contact request handling

### LINKED ISSUE #756: add option to change mail signature
- Platform (linux/mac/windows): Linux
- Delta Chat Desktop Version:  0.102.0-preview
- Expected behavior: possibility to change thi signature similar to the Android client
- Actual behavior: I use for Android a personal signature and would like to use the same in the desktop client but I can't find a way to change it.

### PR DESCRIPTION
**Pull request by Simon-Laux**
closes  #756 & #725

- [x] Translations

### CODE REVIEW

File: _locales/_experimental_en.json [hunk 1]
```diff
@@ -10,5 +10,29 @@
...
+  },
+  "setting_profile": {
+    "message": "Profil"
+  },
+  "setting_profile_displayname": {
+    "message": "Displayname"
+  },
+  "setting_profile_status": {
+    "message": "Signature / Status"
+  },
```
Comment (Jikstra) [inline _locales/_experimental_en.json]: There should be translations in the normal android translations for this, no need to add own/experimental translations

File: src/renderer/components/dialogs/Settings.js [hunk 2]
```diff
@@ -234,6 +242,20 @@ class Settings extends React.Component {
     )
   }
 
+  renderDeltaInput (configKey, label) {
```
Comment (Jikstra) [inline src/renderer/components/dialogs/Settings.js]: Nice!

File: _locales/_experimental_en.json (hunk 1 above)
Comment (Simon-Laux) [reply _locales/_experimental_en.json]: I updated the profile ones, but the other translations aren't pulled in from transifex yet.

**Comment by hpk42** [comment]
nice work @Simon-Laux :)

**Pull request merged**